### PR TITLE
return rawresponse for copy operation

### PR DESCRIPTION
**Pull request by Jinming-Hu**
closes https://github.com/Azure/azure-sdk-for-cpp/issues/1806

# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

File: sdk/storage/azure-storage-files-shares/test/share_file_client_test.cpp [hunk 1]
```diff
@@ -680,6 +680,7 @@ namespace Azure { namespace Storage { namespace Test {
       auto destFileClient
           = m_shareClient->GetRootDirectoryClient().GetFileClient(LowercaseRandomString(10));
       auto copyOperation = destFileClient.StartCopy(fileClient.GetUrl());
+      EXPECT_NE(copyOperation.GetRawResponse(), nullptr);
```
Comment (vhvb1989) [inline sdk/storage/azure-storage-files-shares/test/share_file_client_test.cpp:683]: Why not checking some response code or anything else?

**Review by vhvb1989 — APPROVED** [review]

File: sdk/storage/azure-storage-files-shares/test/share_file_client_test.cpp (hunk 1 above)
Comment (Jinming-Hu) [reply sdk/storage/azure-storage-files-shares/test/share_file_client_test.cpp:683]: because i'm lazy 😞

**Pull request merged**